core/schema-management.md

```diff
@@ -0,0 +1,40 @@
+# Schema Management
+
+## Problem Description
+
+To enable data exchange between business parties involved in a Baseline Workflow (or only Workstep?) (in short, parties), there must be present a set of pre-agreed schemas. 
```

Comment:
Parties is capitalized. Don't need to say, "in short, parties".

Reply:
Is it a set of pre-agreed schemas or a pre-agreed schema [pertaining to that single workstep/workflow...or "schema or schemas".  We should clarify what would constitute the need for multiple schema. Also would help to indicate what would constitute a typical schema that would need to be prearranged.  Separately, we should address what methods are best used to arrange the common schema.  What do translators like Mulesoft bring to this?  Which tools are available to make this easy? Can/should there be schema publishing via counterparty (DIDs?) to make this more standard and automatically discoverable?

Reply:
I was thinking we will have a set of schemas and in each Baseline workstep/flow we will use a subset of them. (off-chain communication, on-chain transactions, etc.)
And I share the other questions you've raised.

Reply:
> I was thinking we will have a set of schemas and in each Baseline workstep/flow we will use a subset of them. (off-chain communication, on-chain transactions, etc.)
> And I share the other questions you've raised.

A lot of the schemas are actually well-known and as such, I don't think this necessarily belongs in the core specification. Organizations know how to marshal to and from well-known data models using whatever tools they currently use.

If you are using something that isn't a known format, a second option that is basically "generic/unchecked" could leverage some schema registry as you describe. For the most by-the-book cases though, a stack does not have to implement this if it knows what schema is supported for any given workstep...

Can call out how this works in the BRI-1 impl and API specification, but it is based on this screenshot, which does associate a model (or generic/unchecked one, which would require some further negotiation not necessarily required by the standard):

![image](https://user-images.githubusercontent.com/161261/116629764-fc7c0a80-a91f-11eb-9861-414eab3cd40e.png)

Reply:
1) is the negotiation and definition of a set of schemas that a workgroup agrees upon outside the scope of the spec? who and what decides informational needs by a dynamic set of participants or into a evolving workgroup?
2) [Un]Marshaling takes care of messaging transparency but would parties not still need a dictionary that represents the schemas they are working with? 

New to these discussions ... learning. Apologies if I am not quite understanding the context.